niviz_rater/validation.py

```diff
@@ -1,47 +1,78 @@
+import os
 import yaml
 import json
-import os
 import yamale
 from yamale.validators import DefaultValidators, Validator
-from bids.config import get_option
 
 SCHEMAFILE = os.path.join(os.path.dirname(__file__), 'data/schema.yaml')
 
 
-def load_json(file):
+def _load_json(file):
     with open(file, 'r') as f:
         result = json.load(f)
     return result
 
 
-def get_entities():
-    configfiles = [
-        load_json(file)['entities']
-        for file in get_option('config_paths').values()
-    ]
+def _get_valid_entities(config_files):
```

Comment:
I would name the input here bids_config_files, so readers will know what's expected to be an input (instead of the qc_spec configuration, etc)